lib/utils/status.js

```diff
@@ -21,11 +17,11 @@ function status(uri, options, logger, useColor) {
     `webpack output is served from ${colors.info(useColor, options.publicPath)}`
   );
 
-  if (contentBase) {
+  if (options.static && options.static.length) {
```

Comment:
Let's improve this `options.static.length > 0`

Reply:
```suggestion
  if (options.static && options.static.length > 0) {
```